chatops/test-hubot-st2-badconfig.bats

```diff
@@ -0,0 +1,105 @@
+
+load '../test_helpers/bats-support/load'
+load '../test_helpers/bats-assert/load'
+
+
+ST2_CHATOPS_ENV_FILE='/opt/stackstorm/chatops/st2chatops.env'
+
+@test "SETUP: setup environment variables for authentication related variables." {
+	`sed -e '/ ST2_API_KEY=/s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_TOKEN= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_USERNAME= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_PASSWORD= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_URL= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`;
+}
+
+@test "exit with error status from unresolved ST2_API_URL." {
+	run eval "("\
+	         " cd /opt/stackstorm/chatops; "\
+	         " ST2_API_URL=http://non-existent:9101/ bin/hubot --test; "\
+	         ")"
+
+	assert_failure
+	assert_equal $status 1
+	assert_output --partial "Failed to retrieve commands from "$ST2_API_URL""
+}
+
+@test "exit with error status from only providing user name for authentication." {
+	run eval "("\
+	         " cd /opt/stackstorm/chatops; "\
```

Comment:
Can we change current working directory only once to avoid running `cd /opt/stackstorm/chatops` on every test? This will simplify code a bit